kas-fleet-manager/deploy-kas-fleet-manager.sh

```diff
@@ -93,7 +40,7 @@ clone_kasfleetmanager_code_repository() {
         ${GIT} fetch && \
         ${GIT} checkout ${KAS_FLEET_MANAGER_BF2_REF} && \
         ${GIT} symbolic-ref -q HEAD && \
-        ${GIT} pull --ff-only)
+        ${GIT} pull --ff-only || echo "Skipping 'pull' for detached HEAD")
     fi
   else
     echo "KAS Fleet Manager code directory does not exist. Cloning it..."
```

Comment:
The token in https://github.com/bf2fc6cc711aee1a0c2a/kas-installer/blob/642f3b88f79918eca5319de236ec2b8d5b4ed4ba/kas-fleet-manager/deploy-kas-fleet-manager.sh#L47 is not needed anymore as the bf2 repo is now public.

Furthermore, I think we can configure fetch/cloning from a branch in fork instead of only working with a main repo.

Reply:
Updated to make the git repository configurable.